nvitop-exporter/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM nvitop:latest
```

Comment:
Just build from an NVIDIA driver image. `pipx` will automatically setup the Python dependencies.

Reply:
Nice!